Additional language for conformance statement

stephenrwalli opened the pull request:
Proposed additional conformance language to support future certification work (cribbed from [https://tools.ietf.org/html/rfc2616](https://tools.ietf.org/html/rfc2616)). 
Signed-off-by: Stephen R. Walli stephen.walli@gmail.com

stephenrwalli committed:
Additional language for conformance statement

Proposed additional conformance language to support future certification work (cribbed from https://tools.ietf.org/html/rfc2616). 
Signed-off-by: Stephen R. Walli <stephen.walli@gmail.com>

stephenrwalli commented:
I meant this as a proposed solution to Issue #[367](https://github.com/opencontainers/runtime-spec/issues/367).

wking commented on README.md:
```diff
@@ -21,6 +21,8 @@ Table of Contents
 
 In the specifications in the above table of contents, the keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
 
+An implementation is not compliant if it fails to satisfy one or more of the MUST or REQUIRED level requirements for the protocols it implements. An implementation that satisfies all the MUST or REQUIRED level and all the SHOULD level requirements for its protocols is said to be "unconditionally compliant"; one that satisfies all the MUST level requirements but not all the SHOULD level requiremen ...
```
“satisfies all the MUST or REQUIRED level” → “satisfies all the MUST and REQUIRED level”?

RobDolinMS commented:
Grammar suggestion: each sentence should be its own line.

stephenrwalli committed:
Update README.md

Cleaned up one sentence per line (sorry), and removed the word "level" to clear up the grammar. I agree it is not particularly helpful and there is no additional semantics to "level" in RFC 2119.

stephenrwalli committed:
Removed conditionally compliant for now.

philips commented:
LGTM

wking commented:
On Fri, Apr 08, 2016 at 03:44:28PM -0700, Brandon Philips wrote:

> LGTM

Already landed via opencontainers/runtime-spec#374.
